day7/sheldon/index.css

```diff
@@ -0,0 +1,410 @@
+    display: none;
+  }
+
+  .portrait__container {
+    color: white;
+  }
+
+  .logo {
+    padding: 32px 25px 42px 25px;
+  }
+
+  .detail__content {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 30px;
+    align-items: center;
+  }
+
+  .detail__ring {
+    margin-right: 40px;
+  }
+
+  svg .time {
+    font-size: 48px;
+    line-height: 74px;
+    letter-spacing: 2px;
+    color: white;
+  }
+
+  svg .period {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  svg .done {
+    fill: #ed882c;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  svg .progress-ring__circle {
+    stroke: rgba(255, 255, 255, 0.1);
+  }
+
+  .progress__length {
+    stroke-dasharray: 150 451;
+    transform: rotate(-90deg);
+    transform-origin: 50% 50%;
+    stroke: #ed882b;
+    stroke-linecap: round;
+  }
+
+  .view__block__title {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .class {
+    font-size: 24px;
+    line-height: 24px;
+  }
+
+  .room {
+    font-size: 16px;
+    line-height: 20px;
+    color: #8c99a6;
+    padding-top: 16px;
+  }
+
+  .prograssbar {
+    width: 300px;
+    padding: 22px 20px 20px 20px;
+    background-color: rgba(255, 255, 255, 0.15);
+    margin-top: 32px;
+    border-radius: 8px;
+  }
+
+  .prograssbar__bar {
+    background-color: hsla(0, 0%, 100%, 0.5);
+    width: 100%;
+    height: 4px;
+    margin-top: 15px;
+    border-radius: 2px;
+  }
+  .prograss {
+    height: inherit;
+    border-radius: 2px;
+    width: 25%;
+    background-color: #fff;
+  }
+
+  .list {
+    color: black;
+    background-color: white;
+  }
+
+  .list__head {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 50px 32px 32px 56px;
+  }
+
+  .head__container {
+    max-width: 90%;
+    width: 100%;
+    margin: auto;
+    border-bottom: 2px solid rgba(0, 0, 0, 0.1);
+  }
+
+  button {
+    display: flex;
+    color: #ea3f58;
+    align-items: center;
+    background-color: #fff;
+    border: 1px solid #f1f2f4;
+    border-radius: 30px;
+    outline: none;
+    box-sizing: border-box;
+  }
+
+  .logOut {
+    margin: 16px 16px 17px 18px;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 19px;
+    box-sizing: border-box;
+  }
+
+  button svg {
+    padding-left: 18px;
+  }
+
+  .school {
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 17px;
+    color: rgba(131, 131, 131, 0.5);
+    margin-bottom: 8px;
+  }
+
+  .list__date {
+    font-weight: 500;
+    font-size: 24px;
+    line-height: 28px;
+    color: black;
+  }
+
+  .green {
+    color: aquamarine;
+  }
+
+  .oncomingclass__class {
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: black;
+  }
+
+  .oncomingclass__room {
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 20px;
+    color: rgba(170, 170, 170, 0.5);
+  }
+
+  .oncomingclass {
+    padding: 50px 32px 32px 0px;
```

Comment:
top and bottom need to have same padding

Reply:
why

Reply:
Figma says otherwise